src/index.html

```diff
@@ -7,8 +7,36 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 name="hederlink" >Moyo header</h1>
+    <header class="header">
+      <a href="#" id="hederlink"
+      class="logo">
+        <img src="logo.png" alt="imeg"
+        class="logo_img">
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item"><a href="#" class="is-active nav-link link_with_line">APPLE</a></li>
```

Comment:
```suggestion
          <li class="nav-item">
            <a href="#" class="is-active nav-link link_with_line">APPLE</a>
          </li>
```
Новый тег - новая линия